### PR TITLE
Cover more command with remote endpoint behavior

### PR DESCRIPTION
**Pull request by ikaneshiro**
**Description of the Pull Request (PR):**

- fixed panic in `verify` command due to incorrect flag lookup
#### Adds remote endpoint behavior to the following:
- `search`
- `sign`
- `key newpair`
- `key search`

If I'm missing any let me know and I will fix that as well.

**This fixes or addresses the following GitHub issues:**

- Related to #3222

**Before submitting a PR, make sure you have done the following:**

- Read the [Guidelines for Contributing](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md), and this PR conforms to the stated requirements.
- Added changes to the [CHANGELOG](https://github.com/sylabs/singularity/blob/master/CHANGELOG.md) if necessary according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added tests to validate this PR and tested this PR locally with a `make testall`
- Based this PR against the appropriate branch according to the [Contribution Guidelines](https://github.com/sylabs/singularity/blob/master/CONTRIBUTING.md)
- Added myself as a contributor to the [Contributors File](https://github.com/sylabs/singularity/blob/master/CONTRIBUTORS.md)

Attn: @singularity-maintainers

### CODE REVIEW

**Comment by gvallee** [comment]
Note: The Travis error as nothing to do with the PR; this is the same problem we are seeing for various PRs over the past few days.

**Review by WestleyK — CHANGES_REQUESTED** [review]
Key newpair is panicking:

```bash
$ ./singularity key newpair 
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x50 pc=0xfceb10]

goroutine 1 [running]:
github.com/sylabs/singularity/cmd/internal/cli.handleKeyFlags(0x1bc2580)
	singularity/cmd/internal/cli/key_push.go:92 +0x160
github.com/sylabs/singularity/cmd/internal/cli.glob..func22(0x1bc2580, 0x1c28e88, 0x0, 0x0)
	singularity/cmd/internal/cli/key_newpair.go:27 +0x2f
github.com/spf13/cobra.(*Command).execute(0x1bc2580, 0xc0000b4170, 0x0, 0x0, 0x1bc2580, 0xc0000b4170)
	singularity/vendor/github.com/spf13/cobra/command.go:766 +0x2ae
github.com/spf13/cobra.(*Command).ExecuteC(0x1bc8220, 0x4, 0xc00038f740, 0x2b)
	singularity/vendor/github.com/spf13/cobra/command.go:850 +0x2bb
github.com/sylabs/singularity/cmd/internal/cli.ExecuteSingularity()
	singularity/cmd/internal/cli/singularity.go:144 +0x3f
main.main()
	singularity/cmd/singularity/cli.go:17 +0x20
```

But the verify works now!

File: cmd/internal/cli/search.go [hunk 1]
```diff
@@ -8,6 +8,7 @@ package cli
 import (
```
Comment (WestleyK) [inline cmd/internal/cli/search.go:8]: The copyright needs to be 2018-2019 in this file.

**Review by WestleyK — COMMENTED** [review]
Last suggestion!

**Review by WestleyK — APPROVED** [review]
Looks good to me! :+1:

**Review by mem — APPROVED** [review]

**Pull request merged**